architecture/networking/routes.adoc

```diff
@@ -597,6 +597,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`*router.openshift.io/haproxy.health.check.interval*`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`*haproxy.router.openshift.io/ip_whitelist*` | Sets a xref:whitelist[whitelist] for the route. |
 |`*haproxy.router.openshift.io/hsts_header*` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`*router.openshift.io/cookie-same-site=Strict*` | Sets SameSite attribute to `Strict`, the cookie will not be sent along with requests initiated by third party websites. |
```

Comment:
Based on what I read in the BZ, I would recommend the following: 

"Sets the `SameSite` attribute to `Strict`. This ensures that the cookie will not be sent along with requests initiated by third party websites." 

What you currently have reads as two independent clauses. It worked when the sentence started with 'When,' but the removal of it made it read oddly to me. I see why you want to start it with 'Sets' though.